templates/genkit/firebase.0.9.0.template

```diff
@@ -0,0 +1,63 @@
+import * as z from "zod";
+
+// Import the Genkit core libraries and plugins.
+import {genkit} from 'genkit';
+import {logger} from 'genkit/logging';
+$GENKIT_CONFIG_IMPORTS
+$GENKIT_MODEL_IMPORT
+
+// From the Firebase plugin, import the functions needed to deploy flows using
+// Cloud Functions.
+import {firebaseAuth} from "@genkit-ai/firebase/auth";
+import {onFlow} from "@genkit-ai/firebase/functions";
+
+// Log debug output to tbe console.
+logger.setLogLevel("debug");
+
+const ai = genkit({
+  plugins: [
+$GENKIT_CONFIG_PLUGINS
+  ],
+});
+
+// Define a simple flow that prompts an LLM to generate menu suggestions.
+export const menuSuggestionFlow = onFlow(
+  ai,
+  {
+    name: "menuSuggestionFlow",
+    inputSchema: z.string(),
+    outputSchema: z.string(),
+    authPolicy: firebaseAuth((user) => {
+      // By default, the firebaseAuth policy requires that all requests have an
+      // `Authorization: Bearer` header containing the user's Firebase
+      // Authentication ID token. All other requests are rejected with error
+      // 403. If your app client uses the Cloud Functions for Firebase callable
+      // functions feature, the library automatically attaches this header to
+      // requests.
+
+      // You should also set additional policy requirements as appropriate for
+      // your app. For example:
+      // if (!user.email_verified) {
+      //   throw new Error("Verified email required to run flow");
+      // }
+    }),
+  },
+  async (subject) => {
+		// Construct a request and send it to the model API.
+    const prompt =
+      `Suggest an item for the menu of a ${subject} themed restaurant`;
+    const llmResponse = await ai.generate({
+      model: $GENKIT_MODEL,
+      prompt: prompt,
+      config: {
+        temperature: 1,
+      },
+    });
+
+		// Handle the response from the model API. In this sample, we just
+		// convert it to a string, but more complicated flows might coerce the
+		// response into structured output or chain the response into another
+		// LLM call, etc.
+    return llmResponse.text;
```

Comment:
```suggestion

    // Handle the response from the model API. In this sample, we just
    // convert it to a string, but more complicated flows might coerce the
    // response into structured output or chain the response into another
    // LLM call, etc.
    return llmResponse.text;
```

Reply:
I'm not seeing that in the current version of the template. I might have fixed it along the way (possibly it was tabs before?)